eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -64,6 +59,18 @@ steps:
         --service="${{ parameters.ServiceDirectory }}" 
         --toxenv=sphinx
 
+  - template: ../steps/set-dev-build.yml
+    parameters:
+      ServiceDirectory: ${{ parameters.ServiceDirectory }}
+      BuildTargetingString: ${{ parameters.BuildTargetingString }}
+
+  - task: PythonScript@0
+    displayName: 'Generate Dev Build Packages'
+    condition: eq(variables['SetDevVersion'],'true')
```

Comment:
Isn't this building the packages twice? How do we ensure they we don't have both sets published to artifacts?